tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -706,19 +708,19 @@ def test_create_pgm_input_transformers(mock_init_array: MagicMock, two_pp_objs,
     "power_grid_model_io.converters.pandapower_converter.PandaPowerConverter._get_pgm_ids",
     new=MagicMock(return_value=pd.Series([0])),
 )
-def test_create_pgm_input_transformers__tap_side():
+def test_create_pgm_input_transformers__default():
     # Arrange
     pp_net: pp.pandapowerNet = pp.create_empty_network()
     pp.create_bus(net=pp_net, vn_kv=0.0)
-    pp.create_transformer_from_parameters(
-        pp_net, 0, 0, 0, 0, 0, 0, 0, 0, 0, tap_neutral=12.0, tap_pos=34.0, tap_side="hv"
-    )
-    pp.create_transformer_from_parameters(
-        pp_net, 0, 0, 0, 0, 0, 0, 0, 0, 0, tap_neutral=12.0, tap_pos=34.0, tap_side="lv"
-    )
-    pp.create_transformer_from_parameters(
-        pp_net, 0, 0, 0, 0, 0, 0, 0, 0, 0, tap_neutral=12.0, tap_pos=34.0, tap_side=None
-    )
+    args = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+    pp.create_transformer_from_parameters(pp_net, *args, tap_neutral=12.0, tap_pos=34.0, tap_side="hv")
+    pp.create_transformer_from_parameters(pp_net, *args, tap_neutral=12.0, tap_pos=34.0, tap_side="lv")
+    pp.create_transformer_from_parameters(pp_net, *args, tap_neutral=12.0, tap_pos=34.0, tap_side=None)
+    tap_pos_trafo = pp.create_transformer_from_parameters(pp_net, *args, tap_neutral=12.0, tap_side="hv")
+    pp_net["trafo"]["tap_pos"][tap_pos_trafo] = np.nan
+    pp.create_transformer_from_parameters(pp_net, *args, tap_neutral=np.nan, tap_pos=34.0, tap_side="hv")
+    pp.create_transformer_from_parameters(pp_net, *args, vector_group=None, shift_degree=30)
```

Comment:
Since panda power doesn't require a shift degree in sets of 30 degrees I'd like to see a test with something in between, e.g. 48 degree to see if we handle this as expected

Reply:
Set the 30 -> 31 and the 60 -> 59, the results should still remain the same

Reply:
Added default test for clocks at arbitrary degree angles.